package.json

```diff
@@ -2,9 +2,17 @@
   "name": "reactjs2020q2",
   "version": "1.0.0",
   "description": "",
-  "main": "src/server/app.js",
+  "main": "src/server/index.js",
   "scripts": {
-    "start": "node src/server/app.js"
+    "build:dev": "webpack",
+    "build:prod": "NODE_ENV=production webpack",
```

Comment:
It will not work on Windows, it's better to use `cross-env` package to avoid OS-dependent problems.

```
"build:prod": "cross-env NODE_ENV=production webpack"
```

Reply:
Tried cross-end, works like a charm. But also found that it can be done by webpack with use of mode (which sets NODE_ENV by himself + has some build-in features). https://webpack.js.org/configuration/mode/
So stopped on this variant (hope you don't mind).